client/app/lib/components/NotificationBar.jsx

```diff
@@ -24,7 +24,7 @@ export default class NotificationBar extends React.Component {
   }
 
   render() {
-    const { notification, ...options } = this.props;
+    const { notification, autoHideDuration = 2000, ...options } = this.props;
```

Comment:
Can I check what this `autoHideDuration` does?

Reply:
Sure It's used to set the duration for the notification bar popup

Reply:
Hmm I'm just wondering if I ever saw this popup in production before haha

Reply:
Yup there is! See below.. It was quite annoying as i havent finished reading it and it was gone so i decided to change it but kept the default as it was set previously..

![image](https://user-images.githubusercontent.com/42570513/135761887-2f200668-1674-47ad-9cda-ba219daaae1d.png)

Reply:
ooo I see, so that's the notification.

Btw I realised it might be better to have it as:

`Failed to delete "Fix Edit with IDLE missing from right-click.pdf", please try again.`

I.e. with the quotation marks.